charts/seed-bootstrap/charts/vpa/templates/admission-controller-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vpa-webhook
+  namespace: garden
+spec:
+  ports:
+    - port: 443
+      targetPort: 8000
+  selector:
+    app: vpa-admission-controller
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: vpa-admission-controller
+  namespace: garden
+  labels:
+{{ toYaml .Values.vpa.labels | indent 4 }}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: vpa-admission-controller
+  namespace: garden
+  labels:
+    app: vpa-admission-controller
+{{ toYaml .Values.vpa.labels | indent 4 }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: vpa-admission-controller
+{{ toYaml .Values.vpa.labels | indent 8 }}
+    spec:
+      serviceAccountName: vpa-admission-controller
+      containers:
+      - name: admission-controller
+        args:
+        - ./admission-controller
+        - --v=4
+        - --stderrthreshold=info
+        - --client-ca-file=/etc/tls-certs/ca.crt
+        - --tls-cert-file=/etc/tls-certs/tls.crt
+        - --tls-private-key=/etc/tls-certs/tls.key
+        image: {{ index .Values.global.images "vpa-admission-controller" }}
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        volumeMounts:
+          - name: vpa-tls-certs
+            mountPath: "/etc/tls-certs"
+            readOnly: true
+        resources:
+          limits:
+            cpu: 200m
+            memory: 500Mi
+          requests:
+            cpu: 50m
+            memory: 200Mi
+        ports:
+        - containerPort: 8000
+      volumes:
+        - name: vpa-tls-certs
+          secret:
+            secretName: vpa-tls-certs
```

Comment:
checksum annotation for this secret is missing

Reply:
done